### DISCUSSION TITLE
Github Student Pack select wrong e-mail

### DISCUSSION THREAD

**Discussion by Xiink** [community/community#55891] [GitHub Education]
### Hi everyone,

Hi..
I am preparing to reverify my Github Student Pack, but I accidentally selected my old school email address, which has already been deactivated. Now, when I try to reverify, it shows the message "You were last verified as a student on May 19, 2023. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits," and I am unable to proceed with the reverification.

Is there any way to initiate the verification process again?

**Comment by samus-aran** [comment]
Hi @Xiink 

Users don't have the ability to deactivate a successful coupon but we can revoke your current coupon for you so that you may re-apply. Would you like us to go ahead and do this for you?
